app/src/main/resources/db/migration/tilbakemeldingsmottak/V0.0.6__hendelse_tabell.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE hendelse
+(
+	id            SERIAL                   NOT NULL PRIMARY KEY,
+	journalpostid VARCHAR(255),
+	oppgaveid     VARCHAR(255),
+	hendelsetype  VARCHAR(255)             NOT NULL,
+	tidspunkt     TIMESTAMP WITH TIME ZONE NOT NULL default (now() at time zone 'UTC')
+);
+
+CREATE INDEX hendelse_journalpostid_idx ON hendelse (journalpostid);
+CREATE INDEX hendelse_oppgaveid_idx ON hendelse (oppgaveid);
```

Comment:
Legge til skript for å fylle opp hendelsesloggen med de eksisterende serviceklagene